go-controller/pkg/cni/helper_linux.go

```diff
@@ -266,6 +266,17 @@ func (pr *PodRequest) ConfigureInterface(namespace string, podName string, ifInf
 
 	ifaceID := fmt.Sprintf("%s_%s", namespace, podName)
 
+	// Find and remove any existing OVS port with this iface-id. Pods can
+	// have multiple sandboxes if some are waiting for garbage collection,
+	// but only the latest one should have the iface-id set.
+	uuids, _ := ovsFind("Interface", "_uuid", "external-ids:iface-id="+ifaceID)
+	for _, uuid := range uuids {
+		if out, err := ovsExec("remove", "Interface", uuid, "external-ids", "iface-id"); err != nil {
```

Comment:
@dcbw if you want to remote the interface itself, then you will need "destroy".

```
# ovs-vsctl --help |grep 'destroy TBL'
  destroy TBL REC             delete RECord from TBL
```

Reply:
@girishmg I wasn't actually intending to destroy the Interface object, just to clear its iface-id from its Interface record. The actual Interface would be destroyed when kubelet/CRI finally called CNI DEL for the sandbox.